scripts/generate_compilation_database

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Follow the same convention as our other scripts if possible (including sourcing the common script).